### ISSUE TITLE
Propagate ol.MapBrowserEvent through $emit

### ISSUE THREAD

**Issue by mfilotto** [tombatossals/angular-openlayers-directive#88]
Hi,

Fist thanks for your work, very handy ;)
It would be great if we had access to the original ol.MapBrowserEvent in our code for singleclick and pointermove events.
I need the mouse position on the screen but want to save an ng-click to do so.

Any chance you add it as a last parameter of you $emit calls ?

**Comment by mfilotto** [comment]
In `olHelpers` file, I replaced 
```javascript
    var setEvent = function(map, eventType, scope) {
        if (eventType === 'pointermove') {
            map.on('pointermove', function(e) {
                var coord = e.coordinate;
                var proj = map.getView().getProjection().getCode();
                if (proj === 'pixel') {
                    coord = coord.map(function(v) {
                        return parseInt(v, 10);
                    });
                    scope.$emit('openlayers.map.' + eventType, {
                        coord: coord,
                        projection: proj
                    });
                } else {
                    scope.$emit('openlayers.map.' + eventType, {
                        lat: coord[1],
                        lon: coord[0],
                        projection: proj
                    });
                }
            });
        } else if (eventType === 'singleclick') {
            map.on('singleclick', function(e) {
                var coord = e.coordinate;
                var proj = map.getView().getProjection().getCode();
                if (proj === 'pixel') {
                    coord = coord.map(function(v) {
                        return parseInt(v, 10);
                    });
                    scope.$emit('openlayers.map.' + eventType, {
                        coord: coord,
                        projection: proj
                    });
                } else {
                    scope.$emit('openlayers.map.' + eventType, {
                        lat: coord[1],
                        lon: coord[0],
                        projection: proj
                    });
                }
            });
        }
    };
```

by 
```javascript
    var setEvent = function (map, eventType, scope) {
        map.on(eventType, function (e) {
            var coord = e.coordinate;
            var relativeCoord;
            var proj = map.getView().getProjection().getCode();
            if (proj === 'pixel') {
                coord = coord.map(function (v) {
                    return parseInt(v, 10);
                });
                relativeCoord = {
                    coord: coord,
                    projection: proj
                };
            } else {
                relativeCoord = {
                    lat: coord[1],
                    lon: coord[0],
                    projection: proj
                };
            }
            scope.$emit('openlayers.map.' + eventType, relativeCoord, e);
        });
    };
```

**Comment by claustres** [comment]
Please see #100, it seems that a similar thing has been integrated.

**Comment by mfilotto** [comment]
Resolved by #96
